PostFreeSurfer/PostFreesurferPipelineLongPrep.sh

```diff
@@ -0,0 +1,433 @@
+#!/bin/bash
+
+# # PreFreeSurferPipeline.sh
```

Comment:
Please change the name.

Reply:
Done. Also updated copyright notices where appropriate. If you think you should be on author list, please let me know.